joomla/joomla-cms
# [4.0][com_messages]: Private message: Tooltip is missing for search filter

Pull Request for Issue #26897 

### Testing Instructions

1. Login to Joomla admin with one user
2. Go to the private message section.
3. Click on filter option

See the new blank filter

### Expected result
This should display the filter for Messages with option
-- All 
-- Read
-- Unread
-- Trashed

![Screenshot from 2019-11-02 12-34-29](https://user-images.githubusercontent.com/17097405/68067493-53297800-fd6d-11e9-98c7-848234661b1c.png)


### Actual result

It is not showing any options in filter.
![68747470733a2f2f6973737565732e6a6f6f6d6c612e6f72672f75706c6f6164732f312f61663266313364663439326538383462666530316131626462396537633435342e706e67](https://user-images.githubusercontent.com/17097405/68067495-5a508600-fd6d-11e9-8681-b6442a2d8c09.png)


### Documentation Changes Required
No

#jpbf19 #codepatch
